docs/features/font.md

```diff
@@ -252,6 +252,33 @@ ClassicEditor
 	.catch( ... );
 ```
 
+### Documents colors
+
+In addition to predefined list of colors, the plugin also includes document colors feature. It lists the colors used in the document and lists them in the order of occurrence. Duplicated colors are not inserted multiple times. By default number of document colors is limited, but you can adjust it (or disable it) using {@link module:font/fontcolor~FontColorConfig#documentColors `fontColor.documentColors`} or {@link module:font/fontbackgroundcolor~FontBackgroundColorConfig#documentColors `fontBackgroundColor.documentColors`} options.
+
+In case the edited document doesn't contain any font color or background colors, the document colors section is skipped.
+
+```js
+ClassicEditor
+	.create( document.querySelector( '#editor' ), {
+		fontColor: {
+			documentColors: 10,
```

Comment:
I don't like a context-less sample like this. Why would anyone change the document colors to 10? Either they changed the number of columns in general (in which it'd be good to show and explain this particular case) and need to change the number of document colors to 10 too (in fact, that should happen automatically – TODO). Or they want to have e.g. two rows. Either way, showing both options here would be preferred.

Reply:
@mlewand checked that `documentColors` default to `columns` so you only need to change this value if you want to have multiple rows. Let's explain exactly that and mention `columns` as well.

Reply:
BTW, I can see in the snippet above (`custom-font-color-and-background-color-options.js`) that we modify both values. Why isn't this reflected anywhere else in this document?

Reply:
I add more examples how to configure document colors section and columns together in "Changing the geometry of the color grid" section.